src/System.Diagnostics.FileVersionInfo/tests/System.Diagnostics.FileVersionInfo.Tests/FileVersionInfoTest.cs

```diff
@@ -107,14 +107,13 @@ public void FileVersionInfo_NonExistentFile_FileNotFound()
         [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "NetFX throws ArgumentException in this case")]
         public void FileVersionInfo_RelativePath_CorrectFilePath()
         {
-            using (new FileStream("kernelbase.dll", FileMode.CreateNew, FileAccess.ReadWrite, FileShare.None, 0x1000, FileOptions.DeleteOnClose))
-            {
-                FileVersionInfo fvi = FileVersionInfo.GetVersionInfo("kernelbase.dll");
-                //File name should be the full path to the local kernelbase.dll, not the relative path or the path to the system .dll
-                Assert.Equal(Path.GetFullPath("kernelbase.dll"), fvi.FileName);
-                //FileDescription should be null in the local kernelbase.dll
-                Assert.Equal(null, fvi.FileDescription);
-            }
+            File.WriteAllText("kernelbase.dll", "bogus kernelbase.dll");
+            FileVersionInfo fvi = FileVersionInfo.GetVersionInfo("kernelbase.dll");
+            //File name should be the full path to the local kernelbase.dll, not the relative path or the path to the system .dll
+            Assert.Equal(Path.GetFullPath("kernelbase.dll"), fvi.FileName);
+            //FileDescription should be null in the local kernelbase.dll
+            Assert.Equal(null, fvi.FileDescription);
+            File.Delete("kernelbase.dll");
```

Comment:
everything above delete needs to be in a try and the delete in a finally so if the test fails the file is deleted.

nit: also can you please put a space after // ...